frontend/app/src/theme/community/tokyonight.ts

```diff
@@ -0,0 +1,77 @@
+import type { Theme } from "../types";
+
+// Colors
+const white = "#FFFFFF";
+const offWhite = "#DFDFDF";
+const black = "#000000";
+const raspberry = "#971E57";
+const darkGray = "#696969";
+const silver = "#C0C0C0";
+const veryDarkGray = "#1e2123";
+const lighterDarkGray = "#101010";
+const txtLight = offWhite;
+const notificationAccent = silver;
+
+export function getTheme(base: Theme): Theme {
+    // it's ok to mutate the theme passed in because it's a clone
+    base.name = "tokyonight";
+    base.label = "Tokyo Night";
+    base.burst = true;
+    base.bg = "radial-gradient(circle, rgba(242,154,216,1) 5%, rgba(10,10,10,1) 70%)";
+    base.bd = darkGray;
+    base.txt = white;
+    base["txt-light"] = txtLight;
+    base.icon.txt = darkGray;
+    base.icon.selected = raspberry;
+    base.button.bg = raspberry;
+    base.button.hv = rasperry;
+    base.button["hv-txt"] = darkGray;
+    base.button.txt = "rgba(255 255 255 / 80%)";
+    base.button.disabled = silver;
+    base.collapsible.closed.header.txt = txt;

```

Comment:
This `txt` value is used in several places but it's not defined. I'm sort of guessing it should be white.